src/config.rs

```diff
@@ -126,7 +123,7 @@ impl StarknetGeneralConfig {
                 eth_fee_token_address: self.starknet_os_config.fee_token_address,
                 strk_fee_token_address: contract_address!("0x0"),
             },
-            vm_resource_fee_cost: self.cairo_resource_fee_weights.clone(),
+            vm_resource_fee_cost: Default::default(), // TODO
```

Comment:
Why did you add this TODO?

Reply:
`cairo_resource_fee_weights` was removed upstream, but `vm_resource_fee_cost` still exists. It's not clear what we should do here...

`cairo_lang` has a `versioned_constants.json` with this definition:
```json
    "vm_resource_fee_cost": {
        "bitwise_builtin": 0.16,
        "ec_op_builtin": 2.56,
        "ecdsa_builtin": 5.12,
        "keccak_builtin": 5.12,
        "n_steps": 0.0025,
        "output_builtin": 0,
        "pedersen_builtin": 0.08,
        "poseidon_builtin": 0.08,
        "range_check_builtin": 0.04
    }
```

The previous values came from `general_config.yml`, which was:
```yml
cairo_resource_fee_weights:
  n_steps: 0.0025
```

removed here: https://github.com/starkware-libs/cairo-lang/commit/efa9648f57568aad8f8a13fbf027d2de7c63c2c0#diff-963ec2e2078343b8f2b0a4f018c8091ab7227b1e32972e4d123621a18ed52f47